hazelcast-jet-s3/src/test/java/com/hazelcast/jet/s3/S3TestBase.java

```diff
@@ -81,17 +81,19 @@ void testSink(String bucketName, String prefix, int itemCount, String payload) {
 
         jet.newJob(p).join();
 
-        try (S3Client client = clientSupplier().get()) {
-            long lineCount = client
-                    .listObjects(req -> req.bucket(bucketName).prefix(prefix))
-                    .contents()
-                    .stream()
-                    .map(o -> client.getObject(req -> req.bucket(bucketName).key(o.key()), toInputStream()))
-                    .flatMap(this::inputStreamToLines)
-                    .peek(line -> assertEquals(payload, line))
-                    .count();
-            assertEquals(itemCount, lineCount);
-        }
+        assertTrueEventually(() -> {
+            try (S3Client client = clientSupplier().get()) {
```

Comment:
do we need to recreate the client each time?

Reply:
yes we don't need that